Games/League of Legends/BETA Client/script.js

```diff
@@ -1,32 +1,61 @@
 include(["Functions", "QuickScript", "CustomInstallerScript"]);
+include(["Functions", "Shortcuts","Wine"]);
 include(["Functions", "Verbs", "sandbox"]);
+include(["Functions", "Verbs", "d3dx9"]);
 include(["Functions", "Verbs", "vcrun2015"]);
 
+// Installs League of Legends and creates two shortcuts.
+//
+// One shortcut called League of Legends which starts the launcher
+// and one shortcut called BETA Client which starts the BETA client
+//
+// Important: 
+// The BETA-Client has to be installed using the launcher.
+// After the upgrade the BETA Client will do the game patching.
+
 new CustomInstallerScript()
     .name("League of Legends BETA Client")
     .editor("Riot Games")
     .applicationHomepage("http://euw.leagueoflegends.com/")
-    .author("Plata")
+    .author("Plata, feanor12")
     .installationCommand(function(wizard) {
-        return {command: "msiexec", args: ["/i", "C://LoL/Lol.EUW.msi","APPDIR=C:\\LoL","/q"]};
+        return {command: "msiexec", args: ["/i", "C://LoL_tmp/Lol.EUW.msi","APPDIR=C:\\LoL","/q"]};
```

Comment:
I think that we should update wine runners so that it can directly run msi files